docs/sources/administration/data-source-management/team_lbac.md

```diff
@@ -0,0 +1,158 @@
+
+Cloud access policies are the access controls from Grafana Cloud, the CAP configured for loki should only to be used to gain read access to the logs.
+
+> "If administrator forget to add rule for a team, what happens?"
+> The teams that does not have a rule applied to it, would be able to query all logs if `query` permissions are setup for their role within Grafana.
+
+**Note:** A user who is part of a team within Grafana without a rule will be able to query all logs if there are role based queriying setup.
+
+#### Best practices
+
+We recommend you only add team LBAC permissions for teams that should use the data source and remove default `Viewer` and `Editor` query permissions.
+
+We recommend for a first setup, setting up as few rules for each team as possible and make them additive and not negated.
+
+For validating the rules, we recommend testing the rules in the Loki Explore view. This will allow you to see the logs that would be returned for the rule.
+
+#### Scenarios
+
+**Scenario 1: One rule setup for each team**
+
+We have two teams, Team A and Team B. Loki access is setup with `Admin` roles to have `Admin` permission only.
+
+- Team A has a rule `namespace="dev"`.
+
+- Team B has a rule `namespace="prod"`.
+
+A user that is part of Team A will have access to logs that match `namespace="dev"`.
+
+A user that is part of Team B will have access to logs that match `namespace="prod"`.
+
+**Scenario 2: Multiple rules setup for one team**
+
+We have two teams, Team A and Team B. Loki access is setup with `Admin` roles having `Admin` permission.
+
+- Team A has rule `cluster="us-west-0", namespace="dev|prod"` configured.
+
+- Team B has rule `cluster="us-west-0", namespace="!prod"` configured.
+
+A user that is only part of Team A will have access to logs that match `cluster="us-west-0" AND (namespace="dev" OR namespace="prod")`.
+
+A user that is only part of Team B will have access to logs that match `cluster="us-west-0" AND namespace!="prod"`.
+
+A user that is part of both Team A and Team B will have access to logs that match `cluster="us-west-0" AND (namespace="dev" OR namespace="prod") OR (is this true?) (cluster="us-west-0" AND namespace!="prod")`.
+
+A user that is **not** part of any Team with `Editor/Viewer` role will not have access to query any logs.
+
+**Important**
+
+A `Admin` user that is part of a Team with will only have access to that teams logs
+
+A `Admin` user that is not part of any Team with `Admin` role will have access to all logs
+
+**Scenario 3: Rules that overlap**
+
+We have two teams, Team A and Team B.
+
+- Team A has a rule `namespace="dev"`.
+
+- Team B has a rule `namespace!="dev"`.
+
+A user in Team A will have access to logs that match `namespace="dev"`.
+
+A user in Team b will have access to logs that match `namespace!="dev"`.
+
+> _NOTE:_ A user that is part of Team A and Team B will have access to all logs that match `namespace="dev"` `OR` `namespace!="dev"`.
```

Comment:
That's basically, all the logs.

Reply:
yeas this is the idea that we had a example of this.